specs/README.md

```diff
@@ -1,84 +1,170 @@
-   regarding spec documents in this folder**.  Specs in this folder
-   are ready for implementation or have already been added to Project Reunion.
-* Create pull requests for new or updated specs when requested to do so
-   as part of an issue thread.
+Designing a new Project Reunion API involves creating a spec for the API, authoring some sample code
+an app might use with the API, getting it reviewed by a group of other API designers, and finally
+merging the spec into `main`.
 
-## Please DO NOT:
+| Who                   | Role                                                                                                                                                     |
+| --------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Author                | Member of the Project Reunion community proposing a new API or change to an existing API                                                                 |
+| Design Representative | Member of the Windows API Design Representatives group responsible for the shape and direction of their component area within Windows or Project Reunion |
+| Reviewer              | Person providing commentary on an API under review                                                                                                       |
 
-* Create unsolicited PRs that add or modify  specs. To propose a new feature
-    please follow the Project Reunion contribution process [described here](../docs/contributor-guide.md)
-    and start by [opening a new issue](https://github.com/microsoft/ProjectReunion/issues/new/choose).
+For additional help, create a new `api-design` tagged issue and assign it to @jonwis for assistance.
 
-## Spec workflow
+### Authoring an API Spec
 
 For Project Reunion team members, the spec and API design workflow is:
 
 1. Ensure your proposal is approved and in plan.
 
-2. Create a new working branch ```user/<username>/<feature>``` or work in
-   your fork of the Project Reunion repo.
+2. Create a new working branch `user/<username>/<feature>` or work in your fork of the Project
+   Reunion repo.
+
+3. Create a new folder for your spec under the `specs` folder: `/specs/<feature>`
+
+4. Author your spec using the [spec template](spec_template.md). Please use relative links for
+   images or other assets in the folder. Be sure to read and follow the commented instructions in
+   the template.
+
+5. Work with your API design representative and iterate on the specification in your branch as much
+   as you see fit.
+
+When your spec is ready for review, your API representative will help guide you through the
+remainder.
+
+**DO** follow the authoring guidelines from [the contributor's guide](../docs/contributor-guide.md).
+
+**DO** follow the spec template guidance for authoring samples and IDL.
```

Comment:
The 'spec template guidance' is commentary embedded inside spec_template.md?

I'd expect given past expect but as worded it's not necessarily clear to someone new
CONSIDER: clarifying